Community/Tdarr_Plugin_DUrEX.js

```diff
@@ -0,0 +1,1000 @@
+/* eslint-disable */
+
+const details = () => {
+  return {
+    id: "Tdarr_Plugin_DUrEX",
```

Comment:
Idk if we want a reference to *that* company in the plugin repository, but that's me.